<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoefftWidget</class>
 <widget class="QWidget" name="CoefftWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelLetter">
     <property name="text">
      <string>a</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>изменяется от</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="valueMin">
     <property name="minimum">
      <double>-32000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>32000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
     <property name="value">
      <double>-1.900000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>до</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="valueMax">
     <property name="minimum">
      <double>-32000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>32000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
     <property name="value">
      <double>1.900000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>с шагом</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="valuesStep">
     <property name="minimum">
      <double>-32000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>32000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
